x-pack/solutions/search/plugins/enterprise_search/public/applications/enterprise_search_content/components/connector_detail/overview.tsx

```diff
@@ -73,6 +83,40 @@ export const ConnectorDetailOverview: React.FC = () => {
             </>
           )
       }
+      {isWaitingOnAgentlessDeployment && (
+        <>
+          <EuiCallOut
+            color="warning"
+            title={
+              <EuiFlexGroup alignItems="center">
+                <EuiFlexItem grow={false}>
+                  <EuiLoadingSpinner />
+                </EuiFlexItem>
+                <EuiFlexItem>
+                  {i18n.translate(
+                    'xpack.enterpriseSearch.content.connectors.overview.agentlessDeploymentNotReadyCallOut.title',
+                    {
+                      defaultMessage: 'Provisioning infrastructure',
+                    }
+                  )}
+                </EuiFlexItem>
+              </EuiFlexGroup>
+            }
+          >
+            <EuiSpacer size="s" />
+            <EuiText size="s">
+              {i18n.translate(
+                'xpack.enterpriseSearch.content.connectors.overview.agentlessDeploymentNotReadyCallOut.description',
+                {
+                  defaultMessage:
+                    "We're setting up the agentless infrastructure to run the connector.",
```

Comment:
Nit: I don't think we typically use sentences like this, I think Elastic uses more technical language in the UI for banners and such.
E.g. "Setting up the agentless infrastructure to run the connector." sounds more Elastic/Kibana to me.
Non blocking of course. WDYT?

Reply:
non blocking but easy fix, lemme change it then :)